src/ngraph/runtime/cpu/pass/cpu_fusion.cpp

```diff
@@ -943,3 +945,38 @@ void ngraph::runtime::cpu::pass::CPUFusion::construct_conv_relu()
     auto m = std::make_shared<pattern::Matcher>(prelu, callback);
     this->add_matcher(m);
 }
+
+void ngraph::runtime::cpu::pass::CPUFusion::construct_sigmoid_multiply()
+{
+    //construct predicate to match sigmoid and tanh
+    auto sigmoid_pred = [](std::shared_ptr<Node> n) {
+        bool result = (std::dynamic_pointer_cast<op::Sigmoid>(n) != nullptr) ||
```

Comment:
in this particular case, it might be cleaner to just return the expression rather than assign it to a result variable.

Reply:
Yep, it was a debugging left over. I'll fix that.